base/deprecated.jl

```diff
@@ -1701,6 +1701,12 @@ export hex2num
 # issue #17886
 # deprecations for filter[!] with 2-arg functions are in associative.jl
 
+@eval GMP @deprecate gmp_version() version()
```

Comment:
I think all of these should have a `false` at the end, since none of them are currently exported?